### DISCUSSION TITLE
CR-CMIP-0-4-0: greenhouse gas concentrations - feedback, observations and issues

### DISCUSSION THREAD

**Discussion by znichollscr** [PCMDI/input4MIPs_CVs#171] [General]
Known issues:

- still no EESC (equivalent effective stratospheric chlorine), but probably could be

**Comment by znichollscr** [comment]
From @vnaik60: Add the normal names for species into the variable attributes e.g. hcc140a should have a long_name of "CH3CCl3".

Me: Yep, I'll use the mapping we already have as the basis: https://input4mips-controlled-vocabularies-cvs.readthedocs.io/en/stable/dataset-overviews/greenhouse-gas-concentrations/#variable-name-mapping

**Reply by vnaik60** [reply]
Thank you!

**Reply by znichollscr** [reply]
@vnaik60 I'm thinking of using the more normal names for the variable names, and just leaving the standard names using these weird names. What do you think?

The affected variables would be:

- new name : current name
-    "c2f6": "pfc116"
-    "c3f8": "pfc218"
-    "c4f10": "pfc3110"
-    "c5f12": "pfc4112"
-    "c6f14": "pfc5114"
-    "c7f16": "pfc6116"
-    "c8f18": "pfc7118"
-    "cc4f8": "pfc318"
-    "ch3ccl3": "hcc140a"

**Reply by znichollscr** [reply]
That would mean we would end up with the below instead

```
netcdf ch3ccl3_input4MIPs_GHGConcentrations_CMIP_CR-CMIP-testing_gnz_175001-202212 {
dimensions:
	time = UNLIMITED ; // (3276 currently)
	lat = 12 ;
	bnds = 2 ;
variables:
	float ch3ccl3(time, lat) ;
		ch3ccl3:standard_name = "mole_fraction_of_hcc140a_in_air" ;
		ch3ccl3:long_name = "ch3ccl3" ;
		ch3ccl3:units = "ppt" ;
		ch3ccl3:cell_methods = "area: time: mean" ;
...
```

**Reply by vnaik60** [reply]
Yes, this sounds good! Thanks.

**Comment by znichollscr** [comment]
Superseded by #235
